netlogo-gui/src/main/app/NotificationBanner.scala

```diff
@@ -75,107 +86,104 @@ class NotificationBanner() extends JPanel with ThemeSync with HoverDecoration {
-            setCaretPosition(0)
-          }
-
-          scrollPane = new JScrollPane(editorPane)
-          scrollPane.setPreferredSize(new Dimension(500, 400))
-          val panel = new JPanel(new BorderLayout())
-          panel.add(scrollPane, BorderLayout.CENTER)
-          val options: Array[AnyRef] = Array(I18N.gui.get("common.buttons.ok"))
-
-          val result = JOptionPane.showOptionDialog(
-            null, panel, I18N.gui.get("dialog.interface.newsNotificationTitle"),
-            JOptionPane.DEFAULT_OPTION, JOptionPane.PLAIN_MESSAGE, null, options, options(0)
-          )
-
-          // Check if OK was clicked (index 0 in options array) and hide the NotificationBanner
-          if (result == 0) {
-            setVisible(false) // Hide NotificationBanner
-            prefs.putInt("lastSeenEventId", jsonObjectList.head.eventId)
+  //alwaysShow allows accessing this from the help menu forces it to be shown, even if it has been viewed before
+  def showJsonInDialog(alwaysShow:Boolean): Unit = {
+
+    val prefs = Preferences.userRoot.node("/org/nlogo/NetLogo")
+    try {if (isShowNeeded() || alwaysShow) {
+        val jsonContent = fetchJsonFromUrl()
+        val formattedString = formatJsonObjectList(JsonObjectList)
+
+        val lastSeenEventId = prefs.getInt(LastSeenEventIdKey, -1); // Returns -1 if "event-id" is not found
+        if (JsonObjectList.head.eventId > lastSeenEventId || alwaysShow) {
+
+          val html = InfoFormatter.toInnerHtml(formattedString)
+
+          if (!jsonContent.trim.isEmpty) {
+            EditorPane = new JEditorPane {
+
+              setDragEnabled(false)
+              setEditable(false)
+              setContentType("text/html")
+              setOpaque(true)
+              setBackground(InterfaceColors.CODE_BACKGROUND)
+              setForeground(InterfaceColors.DEFAULT_COLOR) //Set the font color
+              setText(html)
+              setCaretPosition(0)
+            }
+
+            ScrollPane = new JScrollPane(EditorPane)
+            ScrollPane.setPreferredSize(new Dimension(500, 400))
+            val panel = new JPanel(new BorderLayout())
+            panel.add(ScrollPane, BorderLayout.CENTER)
+            val options: List[String] = List(I18N.gui.get("common.buttons.ok"))
+
+            val optionPane = new CustomOptionPane(this, I18N.gui.get("dialog.interface.newsNotificationTitle"), ScrollPane,
+              options)
+            optionPane.setSize(new Dimension(500, 500))
+
+            if (optionPane.getSelectedOption == "OK") {
+              setVisible(false) // Hide NotificationBanner
+              prefs.putInt("lastSeenEventId", JsonObjectList.head.eventId)
+            }
           }
+        }
       }
-    } catch {
+    }
+
+
+
+  catch {
       case e: Exception =>{
-        OptionDialog.showCustom(this, I18N.gui.get("error.dialog.unknown"), e.getMessage, null)
+        new OptionPane(this, I18N.gui.get("error.dialog.unknown"), e.getMessage, null)
```

Comment:
You should not show a message dialog with no buttons, it's more intuitive and user friendly (and matches the rest of the repo) to include at least an OK button, as you did above.